webpack/assets/javascripts/react_app/components/common/forms/Form.js

```diff
@@ -52,7 +57,7 @@ Form.defaultProps = {
   touched: false,
   disabled: false,
   submitting: false,
-  errorTitle: __('Unable to save'),
+  errorTitle: __('Unable to save. '),
```

Comment:
nitpick: this has extra whitespace, also not sure if the dot is really required (as its a title?)

Reply:
The dot and whitespace is intentional to have nice formatting if there is only [1 error](https://github.com/theforeman/foreman/pull/6499#issuecomment-473653134)

Reply:
that may break during translations, it would be better to do something like `__('Unable to save') + ". "` I think

Reply:
Updated.